content/applications/essentials/activities.rst

```diff
@@ -3,236 +3,332 @@ Activities
-deadline`.
+Create new activity types
+-------------------------
 
-This :guilabel:`Schedule` field information can be altered before the activity is scheduled.
+To create a new :ref:`activity type <activities/types>`, click :guilabel:`New` and a blank
+:guilabel:`Activity Type` form loads.
 
-When all configurations are complete, click :guilabel:`Save`.
+Enter a :guilabel:`Name` for the activity type at the top of the form, then enter the following
+information on the form.
 
-.. image:: activities/schedule-recommended-activity.png
-   :alt: Schedule activity popup with emphasis on recommended activities.
+Activity Settings section
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note::
-   If an activity has the :guilabel:`Chaining Type` set to :guilabel:`Suggest Next Activity`, and
-   has activities listed in the :guilabel:`Suggest` field, users are presented with recommendations
-   for activities as next steps.
+- :guilabel:`Action`: using the drop-dwn menu, select an action associated with this new activity
+  type. Some actions trigger specific behaviors after an activity is scheduled, such as:
 
-Trigger next activity
-~~~~~~~~~~~~~~~~~~~~~
+  - :guilabel:`Upload Document`: if selected, a link to upload a document is automatically added to
+    the planned activity in the chatter.
+  - :guilabel:`Phonecall` or :guilabel:`Meeting`: if selected, users have the option to open their
+    calendar to select a date and time for the activity.
+  - :guilabel:`Request Signature`: if selected, a link to open a signature request pop-up window is
+    automatically added to the planned activity in the chatter.
 
-Setting the :guilabel:`Chaining Type` to :guilabel:`Trigger Next Activity` immediately launches the
-next activity once the previous one is completed.
+  .. note::
+     The actions available to select on an activity type vary, depending on the applications
+     currently installed in the database.
 
-If :guilabel:`Trigger Next Activity` is selected in the :guilabel:`Chaining Type` field, the field
-beneath changes to: :guilabel:`Trigger`. From the :guilabel:`Trigger` field drop-down menu, select
-the activity that should be launched once this activity is completed.
+- :guilabel:`Folder`: this field **only** appears if :guilabel:`Upload Document` is selected for the
+  :guilabel:`Action`.
 
-In the :guilabel:`Schedule` field, choose a default deadline for these activities. To do so,
-configure a desired number of :guilabel:`Days`, :guilabel:`Weeks`, or :guilabel:`Months`. Then,
-decide if it should occur :guilabel:`after completion date` or :guilabel:`after previous activity
-deadline`.
+   Using the drop-down menu, select the :guilabel:`Folder` the document is saved to.
+- :guilabel:`Default User`: select a user from the drop-down menu to automatically assign this
+   activity to the selected user when this activity type is scheduled. If this field is left blank,
+   the activity is assigned to the user who creates the activity.
```

Comment:
This line renders kinda funky :upside_down_face: 
![image](https://github.com/odoo/documentation/assets/23611711/4cc23713-5e9f-43a6-ac70-55e959716ba9)

How about trying to put a blank line between this line and 293 right above?

```suggestion

- :guilabel:`Default User`: select a user from the drop-down menu to automatically assign this
   activity to the selected user when this activity type is scheduled. If this field is left blank,
   the activity is assigned to the user who creates the activity.
```

Reply:
I saw that in my make-fast but couldn't figure out the issue. A extra space didn't help either.  Turns out, I added too much space at the end of the line (early line break) and THAT'S why it did that! Who knew!?